src/glossary.rst

```diff
@@ -2134,6 +2180,14 @@ generic trait
 :dp:`fls_h515f11akr91`
 A :dt:`generic trait` is a :t:`trait` with :t:`[generic parameter]s`.
 
+.. _fls_3Ss6jDgtF1of:
+
+generic type
+^^^^^^^^^^^^
+
+:dp:`fls_Zn2pIsMZoTry`
+A :dt:`generic type` is ??? HELP LUKAS!
```

Comment:
Judging from the sole usage of the term in Variance this seems to be just `a type with a generic parameters`

Reply:
Now defined.